fullscreen.bs

```diff
@@ -109,15 +97,15 @@ its <a>node document</a>'s <a>top layer</a>.
 
   <ol>
    <li><p>If <var>node</var> is its <a>node document</a>'s <a>fullscreen element</a>,
-   <a>exit fullscreen</a> that <a for=/>document</a>.
+   <a>exit fullscreen</a> that <a for=/>document</a> with <var>fullyFlag</var> unset.
 
    <li><p>Otherwise, <a lt="unfullscreen an element">unfullscreen <var>node</var></a> within its
    <a>node document</a>.
   </ol>
 </ol>
 
 <p>Whenever the <a>unloading document cleanup steps</a> run with a <var>document</var>,
-<a>fully exit fullscreen</a> <var>document</var>.
+<a>exit fullscreen</a> <var>document</var> with <var>fullyFlag</var> set.
```

Comment:
Sigh, no, that doesn't work either. I've filed https://github.com/whatwg/fullscreen/issues/82 and ask to declare this out of scope for this refactoring.